Detectors/TRD/base/include/TRDBase/PadPlane.h

```diff
@@ -13,7 +13,7 @@
 
 //Forwards to standard header with protection for GPU compilation
 #include "GPUCommonRtypes.h" // for ClassDef
-
+#include <array>
```

Comment:
@jbarrella could you please remove this include and below instead of `std::array` use `o2::gpu::gpustd::array`? When the code is compiled on the host this is just an alias for `std::array`, see https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/Common/GPUCommonArray.h
Btw: Are you missing the definition of `Local2RowColTime` or did I overlook it?

Reply:
You will also need to include `GPUCommonArray.h` inside `PadPlane.h`

Reply:
Yes, I just realized that this `Local2RowColTime()` function got split into `getPad()` and `getPadRow()` ages ago. Somehow I forgot to clean it up properly. I have removed it now so we no longer need `array`. Sorry about that! Thanks for the help anyway.